### ISSUE TITLE
zips is broken

### ISSUE THREAD

**Issue by almartin82** [almartin82/projection_prep#39]
because it doesn't project saves, all save zscores are NaN, which causes lots of problems.  fix that.

**Comment by drewgriffith15** [comment]
What I do with that is pull in the steamer projection since they do a the SV stat projection, but that is not scripted into any function. It's done after the webscrape, and I am sure you dont want to do two different scrapes...

**Comment by almartin82** [comment]
ahh -- well, actually that's a pretty reasonable solution, I think.  the function could take an optional arg that brings in SV from some other system.  because we have pretty compact functions that do the scraping, it would be pretty simple to drop that in (this is the argument for writing this stuff as a package, I guess

we might actually want to generalize it on the `proj_prep` function - the ability to apply SV projections from some other source.  I have an open issue (#23) to allow a user to apply playing time assumptions to the underlying 'true talent' projected by a projection system.  luke has a nice [writeup](http://mrcheatsheet.com/2016/03/09/2015-best-projections/) on this - a lot of the value from some of the projection systems is their ability to get playing time right.

tldr, I guess I'm thinking the best way to handle this is to write a function to cherry-pick `SV` assumptions from any particular source.  short-term, just patching ZIPS to use steamer or whatever also seems to make sense.
